pkg/trace/api/api.go

```diff
@@ -640,12 +627,11 @@ var killProcess = func(format string, a ...interface{}) {
 // the configuration MaxMemory and MaxCPU. If these values are 0, all limits are disabled and the rate
 // limiter will accept everything.
 func (r *HTTPReceiver) watchdog(now time.Time) {
-	cpu, cpuErr := watchdog.CPU(now)
+	cpu, _ := watchdog.CPU(now)
```

Comment:
I guess we should consider removing watchdog code and options entirely, any reason why we wouldn't do it in this PR?

Reply:
+1 on removing the self-OOM later!